contributors/guide/issue-triage.md

```diff
@@ -4,57 +4,233 @@ weight: 10
+
+### DevStats
+The CNCF has created a [suite of Grafana dashboards and charts](https://devstats.cncf.io/) for collecting metrics related to all the CNCF projects. The [Kubernetes dashboard](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1&refresh=15m) can be used to help SIGs view real-time metrics on many aspects of their workflow, including:
+- [Issue Velocity](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1&from=1587157094179&to=1587758294179&refresh=15m&panelId=8&fullscreen): How quickly issues are resolved
+- [PR Velocity](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1&from=1587157166022&to=1587758366022&refresh=15m&panelId=9&fullscreen): Including PR workload per SIG, PR time to approve and merge, and other data
+
+## Process Pointers and Advice from SIGs
+Several SIGs consistently meet weekly or monthly to triage issues. Here are some details about their processes:
+
+### Running a Triage Meeting: Tips from api-machinery
+The api-machinery SIG has found that triage meetings offer valuable opportunities for newcomers to listen, learn, and start contributing. api-machinery hold triage meetings every Tuesday and Thursday and archive recordings via their [Youtube playlist](https://www.youtube.com/playlist?list=PL69nYSiGNLP21oW3hbLyjjj4XhrwKxH2R); here is an [example](https://www.youtube.com/watch?v=bRptR9vd4S8&list=PL69nYSiGNLP21oW3hbLyjjj4XhrwKxH2R&index=2&t=13s). 
+
+In a typical triage meeting, api-machinery members sort through every issue that they haven't triaged since the previous meeting, using a simple query and issue # to track Open PRs and Issues. They usually then:
+- read through the comments and the code briefly to understand what the issue is about.
+- determine by consensus if it belongs to the api-machinery SIG or not. If not, remove the `sig/api-machinery` label.
+- label other SIGs, if appropriate
+- discuss briefly the technical implications
+- assign people with expertise in the domain to review, comment, reject, etc.
+
+api-machinery has found that consistently meeting on a regular, fixed schedule is key to the success of a triaging effort. More frequent, small meetings are better than infrequent, large meetings, they've found. A few other pointers:
+- We try to balance the load, and ask people if they are okay taking on an issue before assigning it to them
+- We skip issues that are closed
+- We also skip cherrypicks, because we consider that the code change was reviewed in the original PR
+- We ensure participation from the entire SIG and support company diversity. 
+- We use this opportunity to mark ["help needed", "good first issue"](#help-wantedgood-first-issues).
+
+### Triage Guide by cluster-lifecycle
+has developed a [triaging page](/sig-cluster-lifecycle/grooming.md) detailing their process, including the [Milestones](#planning-milestones) stage. Here is a [March 2020 presentation](https://www.youtube.com/watch?v=Q07_PfkNjlw) delivered to the SIG chairs and leads group on their process.
```

Comment:
nit, change the start of the sentence to "The SIG has..." or "The cluster-lifecyle SIG has.." ?

Reply:
Hm, I think those words disappeared somehow -- thanks for the catch